src/app/routes/cpsAsset/getInitialData/processUnavailableMedia/index.test.js

```diff
@@ -105,4 +118,43 @@ describe('processUnavailableMedia', () => {
     const receivedPageData = processUnavailableMedia(pageData);
     expect(receivedPageData).toEqual(pageData);
   });
+
+  it('logs the right code when the media asset is revoked', async () => {
+    const pageData = {
+      metadata: {
+        blockTypes: [EXTERNAL_VPID, 'paragraph', 'heading'],
+      },
+      content: {
+        model: { blocks: [{ statusCode: 404, type: EXTERNAL_VPID }] },
+      },
+    };
+    processUnavailableMedia(pageData);
+    expect(loggerMock.warn).toHaveBeenCalledWith(MEDIA_ASSET_REVOKED);
+  });
+
+  it('logs the right code when the media asset has expired', async () => {
+    const pageData = {
+      metadata: {
+        blockTypes: [EXTERNAL_VPID, 'paragraph', 'heading'],
+      },
+      content: {
+        model: { blocks: [{ statusCode: 410, type: EXTERNAL_VPID }] },
+      },
+    };
+    processUnavailableMedia(pageData);
+    expect(loggerMock.warn).toHaveBeenCalledWith(MEDIA_ASSET_EXPIRED);
+  });
+
+  it('logs the right code when the media metadata is unavailable', async () => {
```

Comment:
How about `it('logs the correct message when the media metadata is unavailable'`?